src/FakeItEasy/RootModule.cs

```diff
@@ -121,7 +121,7 @@ public StringBuilderOutputWriterFactory(Lazy<ArgumentValueFormatter> argumentVal
 
             public StringBuilderOutputWriter Create()
             {
-                return new StringBuilderOutputWriter(new StringBuilder(), this.argumentValueFormatter.Value);

```

Comment:
I must have missed something... You removed the `StringBuilderOutputWriterFactory` class in the previous commit. How did it get back here?

Reply:
Oh! I just relooked at it, and I think that GitHub is showing the commits out of order!!!

```
> git log master.. --oneline --decorate
b382b90c (HEAD -> reduce-nullability, origin/reduce-nullability) Remove nullability warning in FakeObjectCreator
adb22da7 Remove nullability warnings in RootModule
26b72aef Have StringBuilderOutputWriter make its own StringBuilder
dd6eb5d2 Remove nullability warning in ValueProducerSignatureHelper
6955fbab Remove nullability warning in PropertyExpressionHelper
1abef20b Remove nullability warning in BuildableCallRule
2cc035a2 Remove nullability warning in ArgumentConstraintmanagerExtensions
c1fbe7bd Remove nullability warning in SymbolExtensions
ad0faac1 Remove nullability warning in ArgumentConstraintTypeMismatchCodeFixProvider
5183ab00 Remove nullability warning in CastleDynamicProxyGenerator
```

Reply:
It's because dates!!

```
commit adb22da791531a2286b38f3aad130ad8a7b24719                                                                                                                                                                                             Author: Blair Conrad <blair@blairconrad.com>
Date:   Sat Oct 19 12:33:11 2019 -0400

    Remove nullability warnings in RootModule

    Stop using StringBuilderOutputWriter.Factory in ArgumentValueFormatter

commit 26b72aef65165910510648daea099b813139e7d0                                                                                                                                                                                             Author: Blair Conrad <blair@blairconrad.com>
Date:   Sat Oct 19 13:32:52 2019 -0400

    Have StringBuilderOutputWriter make its own StringBuilder
```

I don't know how I managed that. Should I try to redate them?

Reply:
> GitHub is showing the commits out of order!!!

Oh, I forgot about that... The "Commits" tab has just one job, and it can't do it properly!
You can reset the dates when rebasing, using `--ignore-date`, this should put them back in order

Reply:
Cool. Done!

Reply:
Thanks! Makes more sense now